src/Brp.Shared.DtoMappers/Profiles/GeboorteBasisProfile.cs

```diff
@@ -0,0 +1,14 @@
+﻿using AutoMapper;
+using Brp.Shared.DtoMappers.Mappers;
+
+namespace Brp.Shared.DtoMappers.Profiles;
+
+public class GeboorteBasisProfile : Profile
+{
+    public GeboorteBasisProfile()
```

Comment:
In de bewoning-informatie-service is dit Profile ook aanwezig. 

[bewoning-informatie-service : GeboorteProfile](https://github.com/BRP-API/bewoning-informatie-service/blob/feature/%2320-leveren-leeftijd-en-volledige-naam-bewoner/src/Bewoning.Informatie.Service/Profiles/GeboorteProfile.cs)